### ISSUE TITLE
Stuck loading

### ISSUE THREAD

**Issue by whitesm2000** [hwchase17/chat-langchain#77]
Hi. I managed to run this in my local machine, but when I asked a question, what should be parameter for llm, it got stuck.

**Comment by fHachenberg** [comment]
You probably ran it on another than the default port 9000, right? In the index.html the port is hardcoded into the request. Change it there and see what comes on the terminal. There might be issues you just haven't seen yet since the /chat endpoint was never called.

**Comment by whitesm2000** [comment]
I don't think so. Like I said, everything pulled up. Just when I wrote the query and hit the button, it just doesn't move forward. If you use the Langchain version of this same one (on the git hub), it has the same issue. I assumed that was some server issue, so I ran it on my local machine, and it has the same issue. So the problem must be that something is outdated.
